data/announcements/scheduled.yaml

```diff
@@ -30,6 +30,15 @@
 # leave the "announcements" key in place
 announcements:
 
+- name: Dockershim removal
+  startTime: 2022-04-13T00:00:00 # Added in PR: https://github.com/kubernetes/website/pull/32805
+  endTime: 2022-05-19T00:00:00 # Expires 2022-05-19
```

Comment:
We may want to keep this banner up for longer than this; it'll still be worth knowing after the release, and the [release team may delay it](https://github.com/kubernetes/sig-release/discussions/1877).